doc/src/manual/arrays.md

```diff
@@ -49,26 +49,26 @@ or a series of dimension sizes passed as a variable number of arguments.
-| [`reshape(A, dims...)`](@ref)               | an array with the same data as the given array, but with different dimensions.                                                                                                                                                                           |
+| [`zeros(T, dims...)`](@ref)                 | an array of all zeros of type `T`, the type defaults to `Float64` if not specified                                                                                                                                                                       |
+| [`zeros(A)`](@ref)                          | an array of all zeros with the same element type and shape as `A`                                                                                                                                                                                        |
+| [`ones(T, dims...)`](@ref)                  | an array of all ones of type `T`, the type defaults to `Float64` if not specified                                                                                                                                                                        |
+| [`ones(A)`](@ref)                           | an array of all ones with the same element type and shape as `A`                                                                                                                                                                                         |
+| [`trues(dims...)`](@ref)                    | a [`BitArray`](@ref) with all values `true`                                                                                                                                                                                                              |
+| [`trues(A)`](@ref)                          | a `BitArray` with all values `true` and the same shape as `A`                                                                                                                                                                                            |
+| [`falses(dims...)`](@ref)                   | a `BitArray` with all values `false`                                                                                                                                                                                                                     |
+| [`falses(A)`](@ref)                         | a `BitArray` with all values `false` and the same shape as `A`                                                                                                                                                                                           |
+| [`reshape(A, dims...)`](@ref)               | an array with the same data as `A`, but with different dimensions                                                                                                                                                                                        |
```

Comment:
Perhaps "an array with the same data as `A`, but with shape `dims`" or "an array with the same data as `A`, but with dimensions `dims`"?